### ISSUE TITLE
UpCloud doesn’t have 1gbps link speed 

### ISSUE THREAD

**Issue by Dibbyo456** [valentinvichnal/vps-compare-upcloud-linode-digitalocean#4]
UpCloud doesn’t have 1000mbps, they have 500mbps

**Comment by proyb6** [comment]
From UpCloud page, they offer upgradability to 1Gbps to meet customer needs? Have you clarified with UpCloud team?

**Comment by Dibbyo456** [comment]
@proyb6 I confirmed it with support team. They only have 500 mbps public bandwidth & 1gbps private bandwidth. So, basically they have 500mbps only.

**Comment by valentinvichnal** [comment]
@Dibbyo456 you are right, when I signed up my trial was limited to 500Mbps but after my first payment they upgraded the public speed to 1Gbps like @proyb6 said. Maybe this is no longer the case and you need to request the support to do it manually?

I will update the repo with their new 5$ server and mention this bandwidth speed.

**Comment by Dibbyo456** [comment]
@valentinvichnal currently they’re limiting new user to 100mbps after first payment they’re limiting to 500mbps.

**Comment by valentinvichnal** [comment]
@Dibbyo456 Good to know, thanks!

Btw, unless you write a file hosting service network bandwidth hardly an issue.
If you store your files and images on Google Cloud and server your static files from CloudFlare and your node.js server responds only with small requests/sockets, you will ran out of CPU much faster than network bandwidth.
(It is easier and cheaper to store your files in Cloud rather than in VPS)

**Comment by proyb6** [comment]
To clarify, I have confirmed with support team that 1Gbps for public network is available for a cost and depend on use case.

**Comment by valentinvichnal** [comment]
@proyb6 Thanks for the info!
